sources/deny.toml

```diff
@@ -65,6 +65,8 @@ skip-tree = [
     { name = "windows-sys" },
     # schnauzer uses cached, which uses an older version of darling
     { name = "darling", version = "=0.14" },
+    # reqwest brings a set of new dependencies
+    { name = "reqwest", version = "=0.12" },
```

Comment:
this is a pretty unhelpful comment - why are we skipping all of `reqwest`'s dependencies?

Generally we should only do this if we're including multiple versions of a crate, and we would do this to the older one which is presumably less likely to change in a significant way. But here we still only have one copy of `reqwest`.